lib/order.rb

```diff
@@ -0,0 +1,42 @@
+require_relative 'menu'
+require_relative 'sms'
+
+class Order
```

Comment:
I was a bit confused about the use of order. But in the specification, it suggests we should be passing an order list with quantities, and a check sum amount, to test against.

In general it looks like you order class is doing a lot. I got around this by creating a `Takeaway` class that handled the user interface, and called order.